packages/@uppy/aws-s3/src/index.test.js

```diff
@@ -1,3 +1,4 @@
+require('whatwg-fetch')
```

Comment:
hey @goto-bus-stop I'm not sure about this part, but does it look good to you?

Reply:
Hmm I think it shouldn't be necessary because jest already has `fetch`? @jeetiss do you remember why you added this line?

Reply:
yes, because without polyfill tests fail with error `Uncaught ReferenceError: fetch is not defined` and as i understand jest hasn't `fetch`

So strange, jsdom adds Headers class from the Fetch standard, but not `window.fetch`
https://github.com/jsdom/jsdom/blob/master/Changelog.md#1510

Reply:
Ohh yes you're right, I hadn't noticed the log messages before. Thanks for explaining, this is fine with me then 👍